website/docs/docs/collaborate/explore-projects.md

```diff
@@ -0,0 +1,142 @@
+
+## Explore the project’s lineage
+
+dbt Explorer provides a visualization of your project’s <Term id="dag">DAG</Term> that you can interact with. To start, select **Overview** in the left sidebar and click the **Explore Lineage** button on the main (center) section of the page. 
+
+If you don't see the lineage graph immediately, click **Render Lineage**. It can take some time for the graph to render depending on the size of your project and your computer’s available memory. The graph of very large projects might not render so, instead, you can select a subset of nodes by using selectors.
+
+The nodes in the lineage graph represent the project’s resources and the edges represent the relationships between the nodes. Resources like tests and macros display in the lineage within their [resource details pages](#view-resource-details) but not within the overall project lineage graph. Nodes are color-coded and include iconography according to their resource type.
+
+To interact with the lineage graph, you can:
+
+- Hover over any item in the graph to display the resource’s name and type.
+- Zoom in and out on the graph by mouse-scrolling.
+- Grab and move the graph.
+- Click on a resource to highlight its relationship with other resources in your project.
+- [Search and select specific resources](#search-resources) or a subset of the DAG using selectors and lineage (for example, `+[YOUR_RESOURCE_NAME]` displays all nodes upstream of a particular resource).
+- [View resource details](#view-resource-details) by selecting a node in the graph (double-clicking).
+
+
+<Lightbox src="/img/docs/collaborate/dbt-explorer/lineage-v1.gif" title="Explore the lineage graph" />
+
+
+## Search for resources {#search-resources}
+With the search bar (on the upper left of the page or in a lineage graph), you can search using keywords or selectors (also known as *selector methods*). The resources that match your search criteria will display as a table in the main section of the page. When you select a resource in the table, its [resource details page](#view-resource-details) will display.
+
+When using keyword search, dbt Explorer will search through your resources using metadata such as resource type, resource name, column name, source name, tags, schema, database, version, alias/identifier, and package name.
+
+When using selector search, you can utilize the dbt node selection syntax including set and graph operators (like `+`). To learn more about selectors, refer to [Syntax overview](/reference/node-selection/syntax), [Graph operators](/reference/node-selection/graph-operators), and [Set operators](/reference/node-selection/set-operators). 
+
+Below are the selection methods currently available in dbt Explorer. For more information about each of them, refer to [Methods](/reference/node-selection/methods).   
+
+- **fqn:** &mdash; Find resources by [file or fully qualified name](/reference/node-selection/methods#the-file-or-fqn-method).
```

Comment:
I get that the methods, when input in the the command lines, use the : at the end but for the purposes of formatting and preventing them from bumping up against the emdash here, would it make sense to either remove the : or put the commands in like `fqn:` ?

Reply:
ahh, i see where you're coming from. 

it's actually the UI label in the dropdown which is why i used bold instead of code font